src/modals/object/ViewObject.vue

```diff
@@ -188,6 +188,23 @@ import { objectStore, navigationStore, catalogStore } from '../../store/store.js
 													@load="(editor) => markdownEditors[key] = editor"
 													@blur="updateMarkdownValue(key, markdownEditors[key])" />
 
+												<!-- Array properties -->
+												<div v-else-if="getPropertyInputComponent(key) === 'NcTextFieldArray'" class="input-with-icon">
+													<NcTextField
+														ref="propertyValueInput"
+														:value="String(formData[key] !== undefined ? (Array.isArray(formData[key]) ? formData[key].join(',') : formData[key]) : (Array.isArray(value) ? value.join(',') : value || ''))"
+														:type="getPropertyInputType(key)"
+														:placeholder="getPropertyDisplayName(key)"
+														:min="getPropertyMinimum(key)"
+														:max="getPropertyMaximum(key)"
+														:step="getPropertyStep(key)"
+														@update:value="updatePropertyValue(key, $event.split(',').map(item => item.trim()))" />
```

Comment:
Same deal as the first change request
```suggestion
														@update:value="updatePropertyValue(key, $event.split(/ *, */g).filter(Boolean))" />
```